assets/src/pages/room/components/MediaControlButtons.tsx

```diff
@@ -109,7 +109,7 @@ const getAutomaticControls = (
   {
     id: "chat",
     icon: isMobile ? MenuDots : Chat,
-    hover: isSidebarOpen ? "Close the chat" : "Open the chat",
+    hover: isSidebarOpen ? `Close the ${isMobile ? "menu" : "sidebar"}` : `Open the ${isMobile ? "menu" : "sidebar"}`,
```

Comment:
IMO it's legible enough and I don't see any need

Reply:
1. `isMobile ? "menu" : "sidebar"` is duplicated and I believe that's what is meant to be a const
2. Does it even make sense to show hover on mobile? Maybe instead of a conditional string it would be better to not show it at all?